src/storage/src/streaming_table/state_table.rs

```diff
@@ -406,12 +416,67 @@ impl<S: StateStore> StateTable<S> {
         for (pk, row_op) in buffer {
             match row_op {
                 RowOp::Insert(row) => {
+                    if ENABLE_STATE_TABLE_SANITY_CHECK && !self.disable_sanity_check {
```

Comment:
IIRC, the sanity check can fail for some executors. Is this still the case?

Reply:
Yes, in `hash_agg` and `hash_join`, we need to disable this check. This is the issue of executors, maybe fixed in the future.